src/infer.ts

```diff
@@ -257,6 +225,88 @@ function listing(prompt: string, positions: FunctionPositions, info: Array<sdk.F
   }
 }
 
+/**
+ * Returns the flags associated with a type.
+ */
+function which_flags(flags_enum: Record<string, string | number>, value: number): string[] {
+  return Object
+    .keys(flags_enum)
+    .flatMap(k => {
+      const k_int = parseInt(k);
+      return isNaN(k_int)
+        ? []
+        : (value & k_int) !== 0
+          ? [flags_enum[k] as string]
+          : []
+    });
+}
+
+type ObjectTypeInfo = {
+  // The name of the type (not including generic type parameters)
```

Comment:
This comment is wrong now that the get_object_type_info now hooks into the new naming stuff.